test/fixture/class-with-constructor-type-in-comment/expected/my-class.ts

```diff
@@ -0,0 +1,71 @@
+export class TestRef {
```

Comment:
Minor, but can we rename this fixture folder to `test/fixture/class-with-swagger-type-comments` ? 

(This is of course assuming that the type comments are indeed generated by swagger codegen)

Reply:
done